packages/blade/package.json

```diff
@@ -30,50 +34,49 @@
     "razorpay",
     "blade"
   ],
-  "main": "./build/components/index.web.js",
   "exports": {
     "./components": {
       "react-native": {
-        "types": "./build/components/index.native.d.ts",
-        "default": "./build/components/index.native.js"
+        "types": "./components.d.ts",
+        "default": "./build/lib/native/components/index.js"
       },
       "default": {
-        "types": "./build/components/index.d.ts",
-        "development": "./build/components/index.development.web.js",
-        "production": "./build/components/index.production.web.js",
-        "default": "./build/components/index.development.web.js"
+        "types": "./components.d.ts",
+        "development": "./build/lib/web/development/components/index.js",
+        "production": "./build/lib/web/production/components/index.js",
+        "default": "./build/lib/web/production/components/index.js"
```

Comment:
@anuraghazra @kamleshchandnani This will be a breaking change for the [website jest config, ](https://github.com/razorpay/frontend-website/blob/2a1249aa33ca0c2bdc59e35babafc35fac1fbf8f/jest.config.js#L14-L17). They will need to update their config otherwise tests will fail. We need to communicate this with them and add this in changelog.

Reply:
Why is the jest config like that?

Reply:
@anuraghazra They are using an older version of Jest which doesn't support reading package exports, that's why a manual mapping was required. However, since we have a file named `index.js` they might not need this. We will need to check this once.